Library/Homebrew/cask/dsl/appcast.rb

```diff
@@ -3,11 +3,21 @@
 module Cask
   class DSL
     class Appcast
+      ATTRIBUTES = [
+        :configuration,
+      ].freeze
       attr_reader :uri, :parameters
+      attr_reader(*ATTRIBUTES)
 
       def initialize(uri, **parameters)
         @uri        = URI(uri)
         @parameters = parameters
+
+        ATTRIBUTES.each do |attribute|
+          next unless parameters.key?(attribute)
+
+          instance_variable_set("@#{attribute}", parameters[attribute])
```

Comment:
Does this have to be dynamic? I think it would be preferable and easier to follow to hardcode `configuration` here.

Reply:
i'd love to, but i actually have zero clue about ruby, i just copied code from URL.rb until it worked ;)

Reply:
Replace this whole block with `@configuration = parameters[:configuration] if parameters.key?(:configuration)`

Reply:
@MikeMcQuaid thanks! done (and verified with brew style && brew tests)